build-tests-samples/heft-serverless-stack-tutorial/.sst/stage

```diff
@@ -0,0 +1 @@
+prod
```

Comment:
Was this file intentionally checked in? It looks like it's `.gitignore`ed

Reply:
Yes, this one file gets checked in because otherwise the SST toolchain will prompt for it interactively. This problem is tracked by https://github.com/serverless-stack/serverless-stack/issues/1537